alerts/geomodel_alert.json

```diff
@@ -0,0 +1,25 @@
+{
+  "elasticsearch_address": "",
+  "localities": {
+    "es_index": "localities",
+    "valid_duration_days": 1,
+    "radius_kilometres": 50
+  },
+  "events": [
+    {
+      "es_index": "events-*",
```

Comment:
Instead of defining the es index in the config file here like this, we can reference this variable from the alert https://github.com/mozilla/MozDef/blob/master/alerts/lib/alerttask.py#L108